service/Abstractions/Compositions/ExportValidationService.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Threading;
+using Microsoft.KernelMemory.ContentStorage;
+using Microsoft.KernelMemory.Search;
+using System.Linq;
+
+namespace Microsoft.KernelMemory.Compositions;
+
+public class ExportValidationService : IExportValidationService
```

Comment:
I would move this class to Core, trying to avoid having (too much) logic in the Abstractions assembly